### ISSUE TITLE
Tutorial not up to date with version 4.1.3 of nanoc

### ISSUE THREAD

**Issue by godbout** [nanoc/nanoc.ws#149]
The part on [compilation and routing rules](http://nanoc.ws/doc/tutorial/#write-pages-in-markdown) says there are two routes in the Rules file:

```
route '/**/*.{html,md}' do
  if item.identifier =~ '/index.*'
    '/index.html'
  else
    item.identifier.without_ext + '/index.html'
  end
end

route '/**/*' do
  item.identifier.to_s
end
````

but nanoc version 4.1.3 doesn't include 2 rules, just one. The other one has been transformed as a compilation rule. It is explained more deeply in the documentation, but for a first approach I got confused.

**Comment by ddfreyne** [comment]
Good catch—will update!

**Comment by ddfreyne** [comment]
There’s a fix in #155.
